js/source/legacy/solGS/pca.js

```diff
@@ -177,7 +177,8 @@ solGS.pca = {
 				}
 			}
 
-			var protocolId = popDetails.genotyping_protocol_d;
+			var protocolId = protocolId = solGS.genotypingProtocol.getGenotypingProtocolId('pca_div');
```

Comment:
Hey @isaak, is protocolId meant to be duplicated like this?
`var protocolId = protocolId =`
The rest all looks good to me

Reply:
Thanks for catching this, @bellerbrock!